auth-api/src/auth_api/models/task.py

```diff
@@ -95,3 +97,27 @@ def find_by_user_and_status(cls, org_id, status):
         return db.session.query(Task).filter_by(account_id=org_id,
                                                 relationship_type=TaskRelationshipType.USER.value, status=status)\
             .first()
+
+    @classmethod
+    def find_task_by_name(cls, task_name):
```

Comment:
What are these used for?

Reply:
they were filtering functions I made when I didn't really know how to do the filtering for fetch_task. guess I should probably remove those.